### PR TITLE
Delete obsolete FIXME comments

### PR DESCRIPTION
**Pull request by okurz**
FIXME should not stay around in code for all infinity :)

### CODE REVIEW

File: crop.py [hunk 1]
```diff
@@ -43,7 +43,7 @@
     png = filename
     filename = filename[0:len(filename) - len(".png")] + '.json'
     needle = json.loads("""{
-        "tags": [ "FIXME" ],
+        "tags": [ "none" ],
```
Comment (Martchus) [inline crop.py]: Considering the commit title "Delete obsolete FIXME *comments*" I would refrain from doing that change here.

Comment (okurz) [reply crop.py:46]: Yes, good point. I will extract these changes into a separate commit stating that we want to exclude the special word from template strings

Comment (okurz) [reply crop.py:46]: done with https://github.com/os-autoinst/os-autoinst/pull/1377/commits/1daacbbcba7b275b018cc4390e6ad858007f4642

**Comment by okurz** [comment]
We lost Travis CI feedback but tests were actually run and passed: https://travis-ci.org/github/os-autoinst/os-autoinst/builds/669873223

**Comment by okurz** [comment]
Oh, there it is :)

**Pull request merged**